### DISCUSSION TITLE
Yourls + nginx + subfolder

### DISCUSSION THREAD

**Discussion by cowanmax** [YOURLS/YOURLS#3561] [Q&A]
Hi guys!
Please write short and clear installation instructions.
I have installed Yourls to subfolder on my domain.
Backend looks good. But I get a 404 error when following a short link.
Could you advise me what to do?

**Comment by ehthxC7x9D** [comment]
Hey @cowanmax,

Have you checked out https://yourls.org/docs/guide/troubleshooting/frequent-issues#404-instead-of-redirection ? Let me know, thanks.

**Reply by cowanmax** [reply]
I've done it.
BTW i have added:
`    # if YOURLS is installed in a subdirectory, change the path
    # to yourls-loader.php accordingly, eg
   try_files $uri $uri/ /yourls/yourls-loader.php$is_args$args;`

in nginx virtual host config file.
Unfortunately, this did not give any results.
